### PR TITLE
Added temperature sensor to datacom switches.

### PR DESCRIPTION
**Pull request by alangregory**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/discovery/datacom.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: SwitchInfoEntry
+                    value: swTemperature 
+                    num_oid: .1.3.6.1.4.1.3709.3.5.201.1.1.2.1.12.
+                    descr: 'Temperature {{ $index }}'
+                    index: 'swTemperature.{{ $index }}'
+                    low_limit: 1
```
Comment (laf) [inline includes/definitions/discovery/datacom.yaml]: How come you are hard coding the low / high limits? Is it not worth leaving it to be autocalculated?

Comment (alangregory) [reply includes/definitions/discovery/datacom.yaml]: These are values supplied by the manufacturer. Where can i check how values are autocalculated ?

Comment (laf) [reply includes/definitions/discovery/datacom.yaml]: Supplied in the docs but not via snmp? If it's not from snmp then leave it as is. The auto calc is done on the current value so can vary.

Comment (alangregory) [reply includes/definitions/discovery/datacom.yaml]: Yep, no info on mibs.

**Pull request merged**